lambda-cloudformation-custom-resource/src/main/scala/feral/lambda/cloudformation/CloudFormationCustomResource.scala

```diff
@@ -14,23 +14,24 @@
  * limitations under the License.
  */
 
-package feral.lambda.cloudformation
+package feral.lambda
+package cloudformation
 
-import cats.effect._
+import cats.ApplicativeThrow
+import cats.MonadThrow
 import cats.effect.kernel.Resource
 import cats.syntax.all._
-import feral.lambda.cloudformation.CloudFormationCustomResourceHandler.stackTraceLines
+import feral.lambda.Context
 import feral.lambda.cloudformation.CloudFormationRequestType._
-import feral.lambda.{Context, IOLambda}
 import io.circe._
 import io.circe.syntax._
 import org.http4s.Method.POST
-import org.http4s.client.Client
 import org.http4s.circe._
-import org.http4s.client.dsl.io._
-import org.http4s.ember.client.EmberClientBuilder
+import org.http4s.client.Client
+import org.http4s.client.dsl.Http4sClientDsl
 
-import java.io.{PrintWriter, StringWriter}
+import java.io.PrintWriter
+import java.io.StringWriter
 
 trait CloudFormationCustomResource[F[_], Input, Output] {
   def createResource(
```

Comment:
Should we also change `Context` to `Context[F]`, so the `remainingTime` is in `F` instead of `IO`? (I have this compiling locally if you'd just like me to push it up.)

Reply:
Good catch, I forgot about that one! Yes, please push :)